### PR TITLE
Add https documentation

### PR DESCRIPTION
**Pull request by brandonfancher**

### CODE REVIEW

File: doc/https.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# HTTPS
+
+Some web technologies require a secure context when not operating over `localhost`. This includes things link clipboard access and WebRTC--two technologies used within the `FractalSys` applet. For this reason, we need a means of running psinode with an SSL.
```
Comment (James-Mart) [inline doc/https.md:3]: things like* clipboard access

File: doc/https.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Generating certs on local machine
+
+1. Install `mkcert` locally
```
Comment (James-Mart) [inline doc/https.md:7]: What's "local," here? Is it within the container? Or on your host machine and then you copy them into the docker container if that's what you're using?

File: services/user/FractalSys/ui/vite.config.ts [hunk 3]
```diff
@@ -52,11 +52,11 @@ const psibase = (options: Options) => {
             ...httpServerConfig,
             https: {
                 key: fs.readFileSync(
-                    "./psibase.127.0.0.1.sslip.io+1-key.pem",
+                    "../../../../psibase.127.0.0.1.sslip.io+1-key.pem",
```
Comment (James-Mart) [inline services/user/FractalSys/ui/vite.config.ts:55]: Can the vite config read environment variables? Might be nice to set the path to the location of the cert and key as an environment variable and not have to locally touch the vite config.

File: doc/https.md (hunk 2 above)
Comment (brandonfancher) [reply doc/https.md:7]: host machine. I'll clarify.

File: services/user/FractalSys/ui/vite.config.ts (hunk 3 above)
Comment (brandonfancher) [reply services/user/FractalSys/ui/vite.config.ts:55]: Yeah, good call.

**Review by James-Mart — APPROVED** [review]

**Pull request merged**